Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorState.cs

```diff
@@ -123,6 +123,14 @@ public InputActionsEditorState(InputActionsEditorState other, SerializedObject a
             // if the asset would be modified during domain reload.
             serializedObject = asset;
 
+            if (other.Equals(default(InputActionsEditorState)))
```

Comment:
I am not sure about this fix, it might solve this particular problem but.... wouldn't the correct solution to actually introduce an explicit default constructor initialising the struct with valid appropriate initial values for all members so that default(InputActionsEditorState) includes the ones below but also appropriate defaults for all the other fields?

Reply:
Right, but the struct is created by default constructor as part of the Window being created, and you can't override the default constructor for a struct in C# - hence why I said we'd need to swap it to being a class instead if we want that, and I don't know if that would affect anything else

Reply:
You are right, I always forget that. Then I approve this.